simple-ipc.c

```diff
@@ -147,6 +147,10 @@ int ipc_send_command(const char *path, const char *message, struct strbuf *answe
 	DWORD mode = PIPE_READMODE_BYTE;
 	int ret = 0, fd = -1;
 
+	trace2_region_enter("simple-ipc", "send", the_repository);
```

Comment:
Is this only for debugging purposes? Or is there a long-term benefit to having these traces?

Reply:
Debugging. Will remove it.